docs/components/concepts/resource-deletion.md

```diff
@@ -0,0 +1,74 @@
+---
+id: resource-deletion
+title: "Resource deletion"
+description: "Use resource deletion to remove process definitions and decision requirements graphs from Camunda."
+---
+
+You can use resource deletion to remove resources from the Camunda Platform. There are two types of resources that can be deleted:
+
+1. Process definitions
+2. Decision Requirements Graphs (DRG)
+
+You can delete a resource using [Operate](../../components/operate/operate-introduction.md), or by sending
+the [delete resource command](/docs/apis-tools/grpc.md#deleteresource-rpc) to the Zeebe API.
```

Comment:
Can you list the command directly here? Can you link to the Zeebe API?

Reply:
I'm a bit confused about this comment. I did list the command and linked it to the Zeebe API already.

Reply:
As in, instead of hopping into another page, could you also just list the command below this? Would that make the page too crowded?

Reply:
I believe it would be too crowded. It's also not relevant for a majority of the users, as they'd only use the API if they don't use Operate.

Reply:
:thumbs-up: